spec/gruf/error_spec.rb

```diff
@@ -125,4 +125,77 @@
       expect { subject }.to raise_error(GRPC::NotFound)
     end
   end
+
+  describe 'functional test' do
+    let(:id) { 1 }
+
```

Comment:
What do you think about building your client in a `let` var instead of inside each example?

Reply:
It needs to be built in the call itself due to the port being dynamic, and the port is generated by the `run_thing_server` hook each time. Using a `let` var will have rspec cache that across calls and will break the impl.